src/AnalysisServices/AnalysisServices.Dataplane/Commands/Synchronize-AzureASInstance.cs

```diff
@@ -13,58 +13,68 @@
 // ----------------------------------------------------------------------------------
 
 using System;
-using System.Collections.Generic;
 using System.Globalization;
 using System.Linq;
 using System.Management.Automation;
 using System.Net;
 using System.Net.Http;
 using System.Net.Http.Headers;
-using System.Text;
 using System.Threading.Tasks;
 using Microsoft.Azure.Commands.AnalysisServices.Dataplane.Models;
 using Microsoft.Azure.Commands.AnalysisServices.Dataplane.Properties;
 using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
-using Microsoft.WindowsAzure.Commands.Common;
 using Microsoft.WindowsAzure.Commands.Utilities.Common;
 using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
 
 namespace Microsoft.Azure.Commands.AnalysisServices.Dataplane
 {
     /// <summary>
-    /// Cmdlet to log into an Analysis Services environment
+    /// Cmdlet to sync Analysis Services server databases.
     /// </summary>
     [Cmdlet("Sync", ResourceManager.Common.AzureRMConstants.AzurePrefix + "AnalysisServicesInstance", SupportsShouldProcess = true)]
     [Alias("Sync-AzureAsInstance", "Sync-AzAsInstance")]
     [OutputType(typeof(ScaleOutServerDatabaseSyncDetails))]
-    public class SynchronizeAzureAzureAnalysisServer : AzurePSCmdlet
+    public class SynchronizeAzureAzureAnalysisServer : AsAzureDataplaneCmdletBase
     {
-        private static TimeSpan DefaultPollingInterval = TimeSpan.FromSeconds(30);
+        /// <summary>
+        /// Default time interval to wait before first poll for sync status.
+        /// </summary>
+        private static readonly TimeSpan DefaultPollingInterval = TimeSpan.FromSeconds(30);
 
+        /// <summary>
+        /// Default time interval to wait between polls for sync status.
+        /// </summary>
         public static TimeSpan DefaultRetryIntervalForPolling = TimeSpan.FromSeconds(10);
```

Comment:
public should be in front of private by convention. also, does it need to be public? can we use internal?

Reply:
Just noticed that the one above is a readonly. The order is fine.

Reply:
Just noticed that the previous one is a static read only. The order is fine. https://stackoverflow.com/questions/150479/order-of-items-in-classes-fields-properties-constructors-methods

Reply:
The test couldn't see it using internal, so left public.

Reordered all fields, properties, constructors and methods, based on StyleCop guidelines.